packages/orchestrator/lib/clients/processor.ts

```diff
@@ -94,23 +95,44 @@ export class OrchestratorProcessor {
     private async processTask(task: OrchestratorTask): Promise<void> {
         this.abortControllers.set(task.id, task.abortController);
         await this.queue.add(async () => {
+            const span = tracer.startSpan('processor.process.task', {
+                tags: { 'task.id': task.id }
+            });
             try {
                 if (task.abortController.signal.aborted) {
                     // task was aborted while waiting in the queue
+                    logger.info(`task ${task.id} was aborted before processing started`);
                     return;
                 }
                 const res = await this.handler(task);
                 if (res.isErr()) {
-                    await this.orchestratorClient.failed({ taskId: task.id, error: res.error });
+                    const setFailed = await this.orchestratorClient.failed({ taskId: task.id, error: res.error });
+                    if (setFailed.isErr()) {
```

Comment:
before if the call to `orchestratorClient.failed` failed we had no info about it (and the task will eventually expire). At least now we log something